device-types/HPE/FlexNetwork-5130-24G-4SFP+-EI.yml

```diff
@@ -0,0 +1,74 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-24G-4SFP+ EI
+slug: hpe-flexnetwork-5130-24G-4SFPP-EI
+part_number: JG932A
+u_height: 1
+is_full_depth: false
+comments: '[HPE FlexNetwork 5130 EI Switch Series QuickSpecs](https://www.hpe.com/psnow/doc/c04394228.pdf)'
+console-ports:
```

Comment:
Would you be able to add weight and airflow if you can find that information? Thank you!